src/utils/internal.ts

```diff
@@ -1391,6 +1401,22 @@ export function assertTickerValid(ticker: string): void {
   }
 }
 
+/**
+ * @hidden
+ *
+ * Validates a ticker value is valid to be created. Some tickers were created with more relaxed rules
+ */
+export function assertTickerCreatable(ticker: string): void {
```

Comment:
Not a fan of this name. `assertTickerIsValid`?

Reply:
`assertTickerValid ` is already defined, and we need to leave that function as is to support grandfathered tickers.

Reply:
> The new restriction is only on new ticker registrations/create_asset calls.  All other usages of Ticker still have the old validation.

I think `create` captures the intended use case of where this validation should be used.

Reply:
Looking at it, this check can be inlined and I can get rid of this function